x/liquidgov/keeper/proposal.go

```diff
@@ -60,7 +60,7 @@ func (keeper Keeper) DeleteProposal(ctx sdk.Context, proposalID uint64, chain_id
 	store.Delete(types.ProposalKey(proposalID, chain_id))
 }
 
-// GetProposalID gets the highest proposal ID
+// GetProposalID gets the highest ICQ'd proposal ID
```

Comment:
I think there could be multiple concurrent proposals - might want to rename to `GetHighestProposalID`